hassio/src/system/hassio-host-info.ts

```diff
@@ -150,6 +150,17 @@ class HassioHostInfo extends LitElement {
               : ""}
           </div>
           <div>
+            ${this.supervisor.host.disk_life_time
+              ? html` <ha-settings-row>
+                  <span slot="heading">
+                    eMMC Lifetime Used
+                  </span>
+                  <span slot="description">
+                    ${this.supervisor.host.disk_life_time - 10}% -
```

Comment:
I guess it is not 0, as this would be -10 :-)

Reply:
lol such an Apple move.